sympy/core/power.py

```diff
@@ -1080,6 +1080,9 @@ def _eval_derivative(self, s):
         dexp = self.exp.diff(s)
         return self * (dexp * log(self.base) + dbase * self.exp/self.base)
 
+    def _eval_derivative_n_times(self, s, n):
+        return Mul(*[self.base for i in range(self.exp)], evaluate=False)._eval_derivative_n_times(s, n)
```

Comment:
This will fail if self.exp is not a positive integer. This also looks to be an inefficient method.